src/app/auth/confirm-login/confirm-login.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ConfirmLoginComponent } from './confirm-login.component';
+
+describe('ConfirmLoginComponent', () => {
```

Comment:
For component tests, we currently place them alongside all other tests in the `tests` directory (not under `src`).
However, I think we can go without tests for this component, as there is not much complicated logic.

Reply:
@AudreyFelicio can help to delete this test file? The PR can be merged then

Reply:
I have deleted the tests file on this branch. I think this PR can be merged now. @seanlowjk @kkangs0226 @dingyuchen perhaps you can take a second look too ?